live-examples/css-examples/filter-effects/meta.json

```diff
@@ -8,6 +8,78 @@
             "title": "CSS Demo: filter",
             "type": "css"
         },
+        "blur": {

```

Comment:
should we keep alphabetical order here and insert `filter` in among the functions, or did you keep it first on purpose? I'm not sure what is better, your call.